Objects/descrobject.c

```diff
@@ -1637,6 +1609,50 @@ property_clear(PyObject *self)
     return 0;
 }
 
+#include "clinic/descrobject.c.h"
+
+PyTypeObject PyDictProxy_Type = {
```

Comment:
I don't want to do this in this commit. This will be harder to review and will be easier to make an error.